core/validation_schema.py

```diff
@@ -103,6 +103,7 @@
             "Primary Intervention Theme": enums.PrimaryInterventionThemeEnum,
         },
         "non-nullable": [
+            "Postcodes",
```

Comment:
We actually don't want to add this here because we carry out this validation as part of `round_4_specific_validations.validate_locations` - the docstring for that function explains why. Otherwise we will validate against Postcodes when a user hasn't entered single/multiple, which we want to avoid.